build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
+	// http://momentjs.com/docs/#/displaying/
+	return moment(date).utc().format(format);
```

Comment:
Sounds like something that deserves a touch o' research. @erikjung let me know if you're feeling overbooked; I'd be happy to look at this, too, as I am now _quite_ curious.